warp10/src/main/java/io/warp10/script/functions/TYPEOF.java

```diff
@@ -64,6 +66,8 @@
   public static final String TYPE_REALMATRIX = "MATRIX";
   public static final String TYPE_PIMAGE = "PIMAGE";
   public static final String TYPE_PFONT = "PFONT";
+  public static final String TYPE_PSHAPESVG = "PSHAPESVG";
```

Comment:
Could you change this to simply PSHAPE as it really is only a PShape created from an SVG, but there is nothing specific to its origin once it is created.

Reply:
Done. Documentation updated.